nnpackage/schema/circle_schema.fbs

```diff
@@ -221,6 +230,12 @@ table VariantSubType {
   has_rank: bool = false;
 }
 
+enum CompressionType : byte {
+  NONE = 0,
+  //Huffman encoding only
```

Comment:
```suggestion
  // Huffman encoding only
```

Reply:
this comment seems unnecessary as it explains there is only Huffman ...

Reply:
I'll remove this comment